misc/net.minetest.minetest.appdata.xml

```diff
@@ -42,5 +48,11 @@
 	<url type="bugtracker">http://www.minetest.net/development/#reporting-issues</url>
 	<url type="translate">http://dev.minetest.net/Translation</url>
 	<url type="donation">http://www.minetest.net/development/#donate</url>
-	<updatecontact>sfan5@live.de</updatecontact>
-</application>
+	<url type="faq">http://wiki.minetest.net/FAQ</url>
+	<url type="help">http://wiki.minetest.net</url>
+	<provides>
+		<binary>minetest</binary>
+	</provides>
+	<translation type="gettext">minetest</translation>
+	<update_contact>sfan5@live.de</update_contact>
```

Comment:
If you look at the diff carefully you can see that this didn't actually change.

When an `appdata` file was added an update contact was needed and nobody wanted to do it, so my email was put there. Also we can't do a "generic" email on @minetest.net afaik.